cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/Conversion.java

```diff
@@ -271,9 +275,54 @@ public Network convert(ReportNode reportNode) {
         }
 
         CgmesReports.importedCgmesNetworkReport(reportNode, network.getId());
+
+        updateWithAllInputs(network, reportNode);
+
         return network;
     }
 
+    private void updateWithAllInputs(Network network, ReportNode reportNode) {
+        if (!sshOrSvIsIncludedInCgmesModel(this.cgmes)) {
+            return;
+        }
+        this.cgmes.invalidateCaches();
+        this.cgmes.setQueryCatalog("-update");
```

Comment:
Can you introduce 2 String constants for the 2 possible query catalog names?
Maybe something like:
- `QUERY_CATALOG_NAME_INIT = ""`
- `QUERY_CATALOG_NAME_UPDATE = "-update"`

Reply:
The purpose of the 2 constants I was talking about is to avoid errors when calling `setQueryCatalog` (see for instance the "Migration steps" added in the PR description).
So these constants should be public and used in every call to `setQueryCatalog`.

Another possibility is to introduce an enum with these values, but it may be restrictive if someone wants to use the CgmesModel with custom additional query catalogs (but I don't know if it's a plausible scenario).